### ISSUE TITLE
Cordova WKWebVIew plugin does not work on simulator

### ISSUE THREAD

**Issue by stripathix** [ionic-team/cordova-plugin-wkwebview-engine#177]
Created a new project using Cordova commands:

cordova create testapp
Then added following lines in config.xml to install cordova-plugin-wkwebview-engine:

```
<allow-navigation href="http://localhost:8080/*" />
<feature name="CDVWKWebViewEngine">
    <param name="ios-package" value="CDVWKWebViewEngine" />
</feature>
<preference name="CordovaWebViewEngine" value="CDVWKWebViewEngine" />
<engine name="ios" spec="^4.4.0" />
<plugin name="cordova-plugin-whitelist" spec="^1.3.2" />
<plugin name="cordova-plugin-wkwebview-engine" spec="git+https://github.com/driftyco/cordova-plugin-wkwebview-engine.git" />
```
Then Executed:

```
cordova platform add ios
cordova build ios
```
Now if I run this application on simulator then I get this screen: 
<img width="295" alt="ktntm" src="https://user-images.githubusercontent.com/8480351/30267952-cbc83028-9701-11e7-9926-4d054a3c3593.png">

***Note:** This works on real devices but on the simulator it always fails. No Idea what's going on.

Ref: https://github.com/ionic-team/cordova-plugin-wkwebview-engine

cordova: 7.0.1 cordova-ios: 4.4.0

**Comment by troyanskiy** [comment]
You should execute:
```
ionic cordova platform add ios
ionic cordova build ios
```

**Comment by mlynch** [comment]
Works fine on xcode 9/ios 11 simulators. Closing as can't reproduce

**Comment by mlynch** [comment]
Yea on second glance it just looks like the build didn't happen so www/index.html wasn't created

**Comment by stripathix** [comment]
@mlynch I am not able to run the app on the simulator. Using xCode9 and iOS11. It showed build succeeded :-( .

<img width="1148" alt="screen shot 2017-09-22 at 5 03 18 pm" src="https://user-images.githubusercontent.com/8480351/30742937-28ffa1c6-9fb9-11e7-9587-b66ddf3c8772.png">
<img width="1156" alt="screen shot 2017-09-22 at 5 03 32 pm" src="https://user-images.githubusercontent.com/8480351/30742938-290ee032-9fb9-11e7-8709-89886c9fd617.png">

**Comment by troyanskiy** [comment]
@stripathix what exact command did you execute to build?

**Comment by stripathix** [comment]
I am using:

```
cordova platform add ios
cordova build ios
```
Because my project is Cordova directory. Whenever I run **ionic cordova build ios** command I get error **[ERROR] Sorry! ionic cordova build can only be run in an Ionic project directory.**

<img width="1280" alt="screen shot 2017-09-22 at 5 20 15 pm" src="https://user-images.githubusercontent.com/8480351/30743229-5e6ff71a-9fba-11e7-8cb9-49b0ba876227.png">

**Comment by troyanskiy** [comment]
So you can't use the plugin `cordova-plugin-wkwebview-engine` from `Ionic-team`.
There are 2 options for you.
1. Switch you project to ionic project
2. Use the plugin from `apache` https://github.com/apache/cordova-plugin-wkwebview-engine

**Comment by stripathix** [comment]
But still, don't understand why it works on a real device but not on simulator.

Is there any difference in WkWebview plugin from ionic team and plugin by apache? Will it affect performance wise?

Will it be the big task to convert Cordova project directory to ionic directory?

**Comment by troyanskiy** [comment]
> Is there any difference in WkWebview plugin from ionic team and plugin by apache? Will it affect performance wise?

No difference

> Will it be the big task to convert Cordova project directory to ionic directory?

Depends on your project

I think its easier to switch to the plugin from apache.

**Comment by stripathix** [comment]
thanks @troyanskiy I will work on that and get back if face any issue.

**Comment by mlynch** [comment]
This *should* work, are you sure there's a `www/index.html`?

**Comment by stripathix** [comment]
Yes, there is an index.html file once build is completed **platforms/ios/www/index.html** 

<img width="298" alt="screen shot 2017-09-22 at 6 20 10 pm" src="https://user-images.githubusercontent.com/8480351/30745172-c66015c8-9fc2-11e7-8965-4e14b19f6669.png">
<img width="307" alt="screen shot 2017-09-22 at 6 20 22 pm" src="https://user-images.githubusercontent.com/8480351/30745173-c6673c72-9fc2-11e7-9ec5-49d6240a991c.png">

**Comment by stripathix** [comment]
Also, I can confirm this was working with the following environment:

a) Xcode 8.x
b) iOS 10.x
c) Plugin version: git+https://github.com/driftyco/cordova-plugin-wkwebview-engine.git#8409dbb

Now when I have updated to Xcode 9 and using the latest version of the plugin it's breaking in the simulator.

**Comment by manucorporat** [comment]
I can't reproduce this issue. Can you try:
```
rm -rf plaftorms
rm -rf plugins
ionic cordova build ios
```
?

**Comment by kgeoir** [comment]
Hello,
  I am using Xcode 9 and IOS11, when I plug the wkwebview plugin, the browser is enable to load index.html.
![screen shot 2017-09-26 at 9 50 54 am](https://user-images.githubusercontent.com/1089725/30848719-43e4a59c-a2a0-11e7-8429-0582e4a058d5.png)
I did rm -rf platform and plugin then build and still breaking. On my side it breaks on Xcode simulator and device BUT works perfectly when launching with cordova!
Any hint?

**Comment by stripathix** [comment]
Finally, I found the reason for this. I think its the PORT **localhost:8080** on which application is served inside WkWebView.

I am having a docker node container running, listening on 8080. So if I stop my docker container then application starts working on the simulator. 

Is there any way to handle this situation. Like if we can change the PORT# for WkWebView?
